text2emoji.py

```diff
@@ -1,7 +1,7 @@
 def main():
-    string=input()
-    lib1="❗eeeeeeeee➕e➖ee⿠⿡⿢⿣⿤⿥⿦⿧⿨⿩eee🟰e❓e🇦🇧🇨🇩🇪🇫🇬🇭🇮🇯🇰🇱🇲🇳🇴🇵🇶🇷🇸🇹🇺🇻🇼🇽🇾🇿eeeeee🇦🇧🇨🇩🇪🇫🇬🇭🇮🇯🇰🇱🇲🇳🇴🇵🇶🇷🇸🇹🇺🇻🇼🇽🇾🇿"
-    for char in string:
+    strg=input()
```

Comment:
Maybe use a better variable name e.g. `user_input`. People would wonder what `strg` is.

Reply:
ok